v7/_posts/release notes/2019-06-04-ODL-7.6.0.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "ODataLib 7.6.0"
+description: "ODataLib 7.6.0 release notes"
+category: "4. Release Notes"
+---
+
+## Changes in ODataLib 7.6.0 Release ##
+
+## Notes ##
+
+ODataLib 7.6.0 includes the following new features, bug fixes and improvements on top of O:
+
+## Features ##
+
+[[#1476](https://github.com/OData/odata.net/pull/1476)] Support reading/writing Edmx with Version=4.01.
+
+[[#1464](https://github.com/OData/odata.net/pull/1464)] Enable to write the nextlink for the collection of entity reference links.
+
+[[#1459](https://github.com/OData/odata.net/pull/1459)] Write nested entity reference link(s) in request/response.
+
+[[#1454](https://github.com/OData/odata.net/pull/1454)] Support versioned preferences for reading/writing OData prefix.
+
+## Fixed Bugs ##
+
+[[#1451](https://github.com/OData/odata.net/issues/1451)] FindType doesn't work with alias-qualified names.
+
+[[#1455](https://github.com/OData/odata.net/issues/1455)] Remove bogus validation error for EntitySetPath.
+
+[[#1463](https://github.com/OData/odata.net/pull/1463)] Validate the resource type and resource set type in the same inheritance tree.
+
+[[#1465](https://github.com/OData/odata.net/issues/1465)] Support out of line annotations can't target enum member.
+
+[[#1467](https://github.com/OData/odata.net/issues/1467)] Support enum parameters for Uri function.
+
+[[#1318](https://github.com/OData/odata.net/issues/1318)] Unescaped colons in relative Uri cause Invalid URI exception.
+
+[[#1469](https://github.com/OData/odata.net/pull/1469)] Add a validation rule about target of the annotation should be allowed in the AppliesTo of the term.
+
+## Improvements ##
+
+[[#1458](https://github.com/OData/odata.net/pull/1458)] Align resource template and generated code.
+
+[[#1474](https://github.com/OData/odata.net/pull/1474)] Fix tests disabled for large object streaming pull request.
```

Comment:
if we can order by the No, it looks better.

Reply:
Fixed.